### ISSUE TITLE
`attribution` attribute

### ISSUE THREAD

**Issue by larsbuntemeyer** [WCRP-CORDEX/cordex-cmip6-cmor-tables#41]
```json
{
    "attribution": "CORDEX model data is based on downscaling <driving_source_id> driving data produced by <driving_institution_id> using the regional model <source_id>. The driving data is licensed under a Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses)
}
```

**Comment by larsbuntemeyer** [comment]
I think [this](https://certificates.creativecommons.org/cccertedu/chapter/4-4-remixing-cc-licensed-work/) pretty much sums it up for CMIP6:
![grafik](https://user-images.githubusercontent.com/5659125/214402535-a8614273-ba5e-49fb-9821-9ce6709c2b0c.png)

**Comment by gnikulin** [comment]
should be checked with experts

**Comment by durack1** [comment]
Hi folks, the `CC BY 4.0` relaxation (from `CC BY-SA 4.0`) means that the `ShareAlike` clause no longer applies to any of the CMIP6 contributing models - see [CMIP6_source_id_licenses.html](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id_licenses.html). So with the `CC BY 4.0` and CC0` being the only licenses that are associated with CMIP6 output, clause 2 above doesn't apply. By "attribution", I believe this is satisfied if you cite the source data, so for e.g. the DOE E3SM-1-1 historical experiment can be identified by the citation/DOI https://doi.org/10.22033/ESGF/CMIP6.11485

**Comment by larsbuntemeyer** [comment]
@durack1 Thanks! Yes, i just realized the relaxation yesterday! So according to [this](https://creativecommons.org/faq/#can-i-combine-material-under-different-creative-commons-licenses-in-my-work):
![grafik](https://user-images.githubusercontent.com/5659125/214824549-052c31c9-ee22-4ec0-ad46-b9185fdee63b.png)
we are free to choose from any `CC` licenses including non-commercial (`NC`). We have a few institutions that might require that. However, i think we should encourage `CC BY 4.0` also for CORDEX.

**Comment by durack1** [comment]
My recommendation, through our CMIP6 learning is that you want to make the licenses as relaxed as possible from the outset, otherwise you'll also have to go through the after-the-fact relaxation, which is a total pain, and more work than you think. For e.g. if you went a CC0, that would mean that ANYTHING could be done with the data, but there is no responsibility to CORDEX or the data providers after the fact

**Comment by larsbuntemeyer** [comment]
Yes, i agree, thanks for sharing your experiences here. This has really been helpful to tackle my understanding of license issues!
